src/main/java/seedu/address/logic/commands/deliverycommand/DeliveryEditCommand.java

```diff
@@ -34,12 +34,13 @@
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the details of the delivery identified "
             + "by the index number used in the displayed delivery list. "
             + "Existing values will be overwritten by the input values.\n"
+            + "'|' symbol means the user must provide at least one of the fields\n"
```

Comment:
Perhaps this should be after the command usage is shown. Actually maybe it shouldn't be shown at all, and leave it to the user guide for them to know what it means. What do you think?

Reply:
hmmm, now to think of it,i think it should be omitted